### DISCUSSION TITLE
We should provide users with a tool to extract firebase data

### DISCUSSION THREAD

**Discussion by richford** [SwipesForScience/SwipesForScience#117] [General]
Right now, it is not obvious to scientific users how to retrieve and analyze their results. If they're lucky, they find @akeshavan's [braindr-analysis repo](https://github.com/akeshavan/braindr-analysis). Instead, we should also provide a CLI that downloads SFS results from firebase into [tidy](http://vita.had.co.nz/papers/tidy-data.pdf) csv files.

Let's use this discussion to talk about what format the data should be in. New widget types might collect different data so I think we should design a fairly flexible format for data extraction from firebase.

@jyeatman, @akeshavan, @kylesome

**Comment by richford** [comment]
Related to #39

**Comment by richford** [comment]
See also https://github.com/DCAN-Labs/SwipesForScienceToolkit

**Comment by kylesome** [comment]
@richford This sounds like a useful feature. Agree that a CLI would be a good way to go as it could be developed independently from the main app. The same CLI could also help with the install process.

An example could be
```
swipes export [DB_PATH] --format=csv
```
(where `DB_PATH` is the realtime database json path)
The script could download the data and transform into the format provided.
